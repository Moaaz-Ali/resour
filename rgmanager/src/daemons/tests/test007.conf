<?xml version="1.0"?>
<!-- When comapred with test6, the old IP address (192.168.1.2) should
     have a NEEDSTOP while the new IP address should have a NEEDSTART
     flag -->
<cluster>
<rm>
	<resources>
    		<resourcegroup name="test1"/>
		<script name="initscript" file="/etc/init.d/sshd"/>
		<ip address="192.168.1.3" monitor_link="yes"/>
	</resources>
	<resourcegroup ref="test1">
		<ip ref="192.168.1.3"/>
		<script ref="initscript"/>
	</resourcegroup>
</rm>
</cluster>
