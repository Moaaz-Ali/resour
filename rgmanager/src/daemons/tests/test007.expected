=== Resources List ===
Resource type: script
Agent: script.sh
Attributes:
  name = initscript [ primary unique ]
  file = /etc/init.d/sshd [ unique required ]

Resource type: ip
Instances: 1/1
Agent: ip.sh
Attributes:
  address = 192.168.1.3 [ primary unique ]
  monitor_link = yes

Resource type: resourcegroup [ROOT]
Instances: 1/1
Agent: resourcegroup.sh
Attributes:
  name = test1 [ primary unique required ]

=== Resource Tree ===
resourcegroup {
  name = "test1";
  ip {
    address = "192.168.1.3";
    monitor_link = "yes";
  }
  script {
    name = "initscript";
    file = "/etc/init.d/sshd";
  }
}
